src/modules/auth/auth.service.ts

```diff
@@ -82,4 +93,53 @@ export class AuthService {
       role,
     };
   }
+
+  async forgotPassword({ email }: AuthForgotPasswordDto) {
+    const user = await this.authRepo.findOneBy({ email });
+    console.log(user);
```

Comment:
delete

Reply:
already done, but forget to push it to origin